File: DofusSpa/src/app/_models/recipe-consumable.ts
import { Consumable } from "./consumable";

export interface RecipeConsumable{
    id : number,
    ankamaID : number,
    name : string,
    url : string,
    imgUrl : string,
    quantity : number,
    level : number,
    type: string,
    consumableID : number,
    consumable : Consumable
}

File: DofusApi/Models/RecipeConsumable.cs
namespace DofusApi.Models
{
    public class RecipeConsumable
    {
        public int ID { get; set; }
        public int AnkamaID { get; set; }
        public string Name { get; set; }
        public string Url { get; set; }
        public string ImgUrl { get; set; }
        public int Quantity { get; set; }
        public int Level { get; set; }
        public string Type { get; set; }
        public int ConsumableID { get; set; }
        public Consumable Consumable { get; set; }
        
        
    }
}